### PR TITLE
fix: add imageWidth & imageHeight properties on Tile

### PR DESCRIPTION
**Pull request by ClementNumericite**
The proposed change for the Tile component involves adding a feature to specify the width and height of the optional image.  I added imageWidth & imageHeight optionnal properties to follow the imageUrl & imageAlt pattern.

I'll make another suggestion : introduce an imageProps object, similar to the existing linkProps object, which would provide greater flexibility and control over the image properties. 

I am open to discussing this further.

### CODE REVIEW

**Comment by ClementNumericite** [comment]
More context : my Lighthouse CI is throwing error when images have no image & height properties. 
So I can't use the Tile component for now.

[link to the LH report with the errors](https://storage.googleapis.com/lighthouse-infrastructure.appspot.com/reports/1681463154967-66745.report.html)

**Comment by ClementNumericite** [comment]
More context : my Lighthouse CI is throwing error when images have no image & height properties. 
So I can't use the Tile component for now.

[link to the LH report with the errors](https://googlechrome.github.io/lighthouse/viewer/?gist=1b39adc4361931b522beb57447754a96)

**Comment by garronej** [comment]
Works for me 👍🏻   
Thanks for the contribution.

File: src/Tile.tsx [hunk 1]
```diff
@@ -15,6 +15,8 @@ export type TileProps = {
     desc?: ReactNode;
     imageUrl?: string;
     imageAlt?: string;
+    imageWidth?: string | number;
```
Comment (garronej) [inline src/Tile.tsx:18]:
```suggestion
    /** Default: undefined */
    imageWidth?: string | number;
```

File: src/Tile.tsx [hunk 2]
```diff
@@ -15,6 +15,8 @@ export type TileProps = {
...
+    imageHeight?: string | number;
```
Comment (garronej) [inline src/Tile.tsx:19]:
```suggestion
    /** Default: undefined */
    imageHeight?: string | number;
```

**Pull request merged**